Cargo.toml

```diff
@@ -27,3 +27,8 @@ backtrace = "0.3"
 
 [features]
 nightly = []
+
+[workspace]
+members = [
+  "tests/simple",
```

Comment:
Instead of "simple" having a clearer specifier might be useful. Having "simple" as a category isn't very clear on what should be inside of it.